bokehjs/src/coffee/models/glyphs/image_rgba.ts

```diff
@@ -8,6 +8,11 @@ import {Context2d} from "core/util/canvas"
 export class ImageRGBAView extends XYGlyphView {
   model: ImageRGBA
 
+  initialize(options: any): void {
+    super.initialize(options);
+    this.connect(this.model.properties.global_alpha.change, () => this.renderer.request_render());
+  }
+
```

Comment:
I am not sure this is necessary, I would expect it to be covered by this:

https://github.com/marcuskd/bokeh-fork/blob/b6c21f35195ec6c7a9c826400a9598034eb77a37/bokehjs/src/coffee/models/renderers/glyph_renderer.ts#L118

@marcuskd do you know for certain that changing alpha does not result in an update without this line? If not can you test with it removed?

Reply:
I added this to Image (in PR#7570) and ImageRGBA as it was already in ImageURL so I assumed it was necessary. I'll remove it and test.

Reply:
Ok thanks! It may be needed, but it's worth double checking.

Reply:
@bryevdv no, that line was not necessary. I'll remove it from all three image types and push the changes shortly...

Reply:
Thanks!

Reply:
@marcuskd Hrmm I just realized why those are there, the other line I linked does a full `set_data` on changes, but the line just requests a render. In theory that should be a performance optimization. It might be worth just keeping.

Reply:
@bryevdv is the 'initialize' block (see below) needed? I'm not sure what it does but it was already in ImageURL, and my current PR will add it to ImageRGBA.

  initialize(options: any): void {
    super.initialize(options);
  }

Reply:
@marcuskd yes apparently the initialize blocks are necessary for TypeScript reasons.

Reply:
OK, will change back shortly...

Reply:
great sorry for the back and forth and thanks for your patience!